### PR TITLE
Add container messages

### PR DESCRIPTION
**Pull request by paulotijero**
It was built the container messages, contain and was resolve the following:
- [x] Use notification
- [x] User messages 
- [x] Line date
- [ ] chat footer

### CODE REVIEW

File: public/assets/css/style.css [hunk 1]
```diff
@@ -239,3 +240,138 @@ body {
...
+}
+
+.-b-top {
+  border-top: 1px solid hsla(0, 0%, 100%, 0.06);
+}
+
+.-b-bottom {
+  border-bottom: 1px solid hsla(0, 0%, 100%, 0.06);
+}
+
```
Comment (diegotc86) [inline public/assets/css/style.css]: `.-b-bottom` is no longer in use. Evaluate if is necessary to keep.

Comment (paulotijero) [reply public/assets/css/style.css]: You right, I forget to erase :sweat_smile: now resolve.

File: public/assets/css/style.css [hunk 2]
```diff
@@ -239,3 +240,138 @@ body {
...
+.line::after,
+.line::before {
+  content: " ";
+  width: calc(100% - 300px);
+  border-bottom: 1px solid hsla(0, 0%, 100%, 0.06);
+}
+
```
Comment (Linzeur) [inline public/assets/css/style.css]: Oh, I get the way how to use these pseudo classes, I suppose the date will be in the middle of them. Good idea

File: public/index.html [hunk 3]
```diff
@@ -131,15 +143,87 @@
...
+              <span class="date">hoy a las 14:51</span>
+            </li>
+            <li class="container-messages -b-top">
+              <div class="user-img">
+                <img
+                  src="https://discordapp.com/assets/0e291f67c9274a1abdddeb3fd919cbaa.png"
+                  alt="user-image"
+                />
+              </div>
+              <div class="user-messages">
```
Comment (Gatts1) [inline public/index.html]: I like this block. Good job!

File: public/assets/css/style.css (hunk 2 above)
Comment (paulotijero) [reply public/assets/css/style.css]: Yes, you right, the date is in the middle of them.

File: public/index.html [hunk 4]
```diff
@@ -131,15 +143,87 @@
...
+          <ul>
+            <li class="line">
+              <span class="line-date">23 de mayo de 2019</span>
+            </li>
+            <li class="container-notification">
+              <svg><use xlink:href="#arrow-right"/></svg>
+              <span class="group-notification">Un <span class="user">Gatts</span> salvaje apareció.</span>
+              <span class="date">hoy a las 14:51</span>
+            </li>
+            <li class="container-notification -b-top">
```
Comment (diegotc86) [inline public/index.html]: We will use this html template for generating the messages dinamically

File: public/assets/css/style.css (hunk 2 above)
Comment (diegotc86) [reply public/assets/css/style.css]: Can you explain en an easy way how `::before` and `::after` works?

**Pull request merged**